### PR TITLE
wip: add script for embedding new kernel

### PR DESCRIPTION
**Pull request by Williangalvani**
helps #685

### CODE REVIEW

**Review by patrickelectric — CHANGES_REQUESTED** [review]
I just tested and it appears that is not generating a working image for the pi 4.

**Comment by patrickelectric** [comment]
When building locally the img files just fails to run on the raspberry.
When using the following dockerfile, it just fails to provide the .img with the correct files.
```
FROM debian:bullseye-slim

#COPY 2021-10-30-raspios-bullseye-armhf-lite.img 2021-10-30-raspios-bullseye-armhf-lite.img
RUN apt update && apt install -y unzip xz-utils tar wget
RUN wget https://developer.arm.com/-/media/Files/downloads/gnu-a/10.3-2021.07/binrel/gcc-arm-10.3-2021.07-x86_64-arm-none-linux-gnueabihf.tar.xz
RUN tar -xvf gcc-arm-10.3-2021.07-x86_64-arm-none-linux-gnueabihf.tar.xz
ENV PATH="$PWD/gcc-arm-10.3-2021.07-x86_64-arm-none-linux-gnueabihf/bin:${PATH}"

COPY setup.sh /usr/bin/setup.sh
COPY build.sh /usr/bin/build.sh
RUN chmod +x /usr/bin/setup.sh
RUN chmod +x /usr/bin/build.sh

RUN apt update
RUN apt install -y bc build-essential cpio fakeroot libssl-dev ncurses-dev git make sudo bison flex kmod
RUN mkdir -p /tmp/run
CMD cd /tmp/run && setup.sh
```

**Comment by patrickelectric** [comment]
It would be nice to have a reliable way to create such image

**Comment by Williangalvani** [comment]
should we deplot this to s3 or something? too bad we can't reliably install docker in CI yet.

File: deploy/initial-setup/setup.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+FILE=raspios_lite_armhf-2021-11-08/2021-10-30-raspios-bullseye-armhf-lite.zip
+FILENAME=$(basename $FILE)
+wget -nc https://downloads.raspberrypi.org/raspios_lite_armhf/images/$FILE
+unzip -n "$FILENAME"
+
+git clone --depth 1  --branch rpi-4.19.y https://github.com/Williangalvani/linux
```
Comment (patrickelectric) [inline deploy/initial-setup/setup.sh:7]: Can we use the official linux repository + branch and add the navigator file here ?
I have done it in https://github.com/bluerobotics/companion-docker/pull/705 as well, but I just saw the rework of the docker file on this PR now.

Comment (patrickelectric) [reply deploy/initial-setup/setup.sh:7]: It would be nice to have a readme with instructions as well.

Comment (Williangalvani) [reply deploy/initial-setup/setup.sh:7]: > Can we use the official linux repository + branch and add the navigator file here ? I have done it in #705 as well, but I just saw the rework of the docker file on this PR now.

yep, I think that is the way to go. I'll do it.

**Comment by Williangalvani** [comment]
we realized that we can use the stock kernel now.

**Pull request closed without merging**